pr_agent/agent/pr_agent.py

```diff
@@ -15,6 +15,7 @@
 from pr_agent.tools.pr_config import PRConfig
 
 command2class = {
+    "auto_review": PRReviewer,
```

Comment:
**Suggestion:** Consider adding a docstring to the 'auto_review' command in the 'command2class' dictionary to explain its purpose and usage.
```suggestion
    "auto_review": PRReviewer,  # This command triggers an automatic review of the PR
```